bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYJets_HT_LO_MLM/DYJetsToMuMu_HT_mll105_160/DYJetsToMuMu_HT_mll105_160_proc_card.dat

```diff
@@ -0,0 +1,39 @@
+#************************************************************
+#*                        MadGraph 5                        *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 1.5.11                2013-06-21         *
+#*                                                          *
+#*    The MadGraph Development Team - Please visit us at    *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph 5                *
+#*                                                          *
+#*     run as ./bin/mg5  filename                           *
+#*                                                          *
+#************************************************************
+
+import model sm-ckm_no_b_mass
+define p = g u c d s u~ c~ d~ s~
```

Comment:
naive question: aren't you missing the b quark here?

Reply:
@perozzi, I actually used these ones as examples: https://github.com/cms-sw/genproductions/tree/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/DYJets_HT_LO_MLM/DYJets_HT_mll50/DYJets_HT-100to200
but I think you are right, we should have a b jets there since it is asked the 5-flav scheme.
Were these cards the one used for the 2016 Moriond17 production?

Reply:
I checked [*] and am afraid so........
(@bendavid can you have a look too?)

[*] https://cms-pdmv.cern.ch/mcm/public/restapi/requests/get_fragment/SUS-RunIIWinter15wmLHE-00030
pointing to
/cvmfs/cms.cern.ch/phys_generator/gridpacks/slc6_amd64_gcc481/13TeV/madgraph/V5_2.2.2/DYJets_HT_LO_MLM/DYJets_HT-100to200/V1/DYJets_HT-100to200_tarball.tar.xz
for which the log contains

Defined multiparticle p = g u c d s u~ c~ d~ s~
[...]
generate p p > l+ l- / h @0

Reply:
@kdlong @vciulli @qliphy can you have a look too and comment please?

Reply:
Any news ? I can change and produce the one with mll 105-160 with a b inside, but you may want to understand if this is correct before the new production.